crates/cheatnet/src/runtime_extensions/call_to_blockifier_runtime_extension/panic_data.rs

```diff
@@ -1,14 +1,13 @@
-use conversions::{byte_array::ByteArray, felt::FromShortString};
+use conversions::byte_array::ByteArray;
 use regex::Regex;
 use starknet_types_core::felt::Felt;
 
 #[must_use]
 pub fn try_extract_panic_data(err: &str) -> Option<Vec<Felt>> {
-    // FIXME add support for comma separated errors
-    let re_felt_array = Regex::new(
-        r"[\s\S]*Execution failed\. Failure reason:\nError in contract \(.+\):\n.*\('([\s\S]*)'\).",
-    )
-    .expect("Could not create felt panic_data matching regex");
+    // Matches panic data formatted with:
+    // https://github.com/starkware-libs/sequencer/blob/8211fbf1e2660884c4a9e67ddd93680495afde12/crates/starknet_api/src/execution_utils.rs
+    let re_felt_array = Regex::new(r"(0x[a-fA-F0-9]+)\s\('[^']*'\)")
```

Comment:
I like the simplification, however I think we should still be catching the `Got an exception while executing a hint...` prefix. Otherwise we are risking mapping any error containing parsable felts to our format, which I think could be problematic in some weird cases.